internal/handlers/common/update.go

```diff
@@ -76,14 +78,16 @@ func UpdateDocument(doc, update *types.Document) (bool, error) {
 
 				if !doc.Has(incKey) {
 					must.NoError(doc.Set(incKey, incValue))
-					return true, nil
+					changed = true
```

Comment:
@AlekSi @w84thesun  here influences on $inc operator - some cases are uncovered, for example when there are several `$inc` in the update document.  WDYT?

Reply:
I agree, but that (and the one above) change in logic looks suspicious; let's add tests for that

Reply:
nope, sorry not for this. adding...

Reply:
> nope, sorry not for this. adding...

nope, no tests needed to add here - it's already covered.